ste/sender-azureFile.go

```diff
@@ -164,13 +200,22 @@ func (u *azureFileSenderBase) GetDestinationLength() (int64, error) {
 	return prop.ContentLength(), nil
 }
 
+func (u *azureFileSenderBase) EnsureFolderExists() error {
+	return AzureFileParentDirCreator{}.CreateDirToRoot(u.ctx, u.dirURL(), u.pipeline, u.jptm.GetFolderCreationTracker())
+}
+
+func (u *azureFileSenderBase) SetFolderProperties() error {
+	// TODO
+	return nil
```

Comment:
Is there no properties or metadata that need to be transferred S2S already?

Or is this left as a TODO because it might be better to implement when handling the remainder of SDDLs and SMB properties?

Reply:
Yes, the latter.  We can do it in the followup PR